package.json

```
@@ -116,7 +116,7 @@
     "start:companion": "bash bin/companion.sh",
     "start:companion:with-loadbalancer": "e2e/start-companion-with-load-balancer.mjs",
     "build:bundle": "yarn node ./bin/build-bundle.mjs",
-    "build:clean": "git clean -e node_modules -Xfd packages e2e .parcel-cache coverage",
+    "build:clean": "cp .gitignore .gitignore.bak && grep -v 'node_modules' .gitignore.bak > .gitignore && echo '!**/node_modules/**/*' >> .gitignore; git clean -Xfd packages e2e .parcel-cache coverage; mv .gitignore.bak .gitignore",
```

Actually we don't need `grep -v`
```suggestion
    "build:clean": "cp .gitignore .gitignore.bak && printf '!node_modules\n!**/node_modules/**/*\n' >> .gitignore; git clean -Xfd packages e2e .parcel-cache coverage; mv .gitignore.bak .gitignore",
```